.zuul.yaml

```diff
@@ -3,26 +3,46 @@
     name: packit-service/ogr
     check:
       jobs:
-        - tests
-        - pre-commit
+        - ogr-pre-commit
```

Comment:
wouldn't `pre-commit` be enough?

Reply:
The zuul needs identical job names accross all projects.

As you can see in the first comment by zull that is hidden:

```
Job tests-rpm in packit-service/packit is not permitted to shadow job tests-rpm in packit-service/ogr
```

So I added prefix to all jobs.

Reply:
Ok, I see. So this is resolved then.